lib/qa/authorities/linked_data/config/search_config.rb

```diff
@@ -0,0 +1,157 @@
+module Qa::Authorities
+  module LinkedData
+    module SearchConfig
```

Comment:
Provide module documentation about what this is for and where we should be using it.  Are all of these public methods?  Could this have been a class rather than a mixin?